src/cyw43.h

```diff
@@ -321,6 +321,17 @@ int cyw43_wifi_join(cyw43_t *self, size_t ssid_len, const uint8_t *ssid, size_t
  */
 int cyw43_wifi_leave(cyw43_t *self, int itf);
 
+/*!
+ * \brief Get the signal strength (RSSI) of the wifi network
+ *
+ * For STA (client) mode, returns the signal strength or RSSI of the wifi network.
+ * An RSSI value of zero is returned if you call this function before a network is connected.
+ *
+ * \param self the driver state object. This should always be \c &cyw43_state
+ * \return 0 on success
+ */
+int cyw43_get_rssi(cyw43_t *self, int *rssi);
```

Comment:
I think this should be called `cyw43_wifi_get_rssi()`.  And the `rssi` argument should be `int32_t` to be clear that it's 32-bit.

Reply:
Rebased and fixed.